### PR TITLE
Add ClientJS@0.1.11 / npm auto-update

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #10295

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/jackspirou/clientjs
 * Official website (optional, not the repository): https://clientjs.org/
 * NPM package url (optional): https://www.npmjs.com/package/clientjs
 * License and its reference: [Apache 2.0](https://github.com/jackspirou/clientjs/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 12
   - Count of stars: 216
   - Count of forks: 58
 * NPM download stats (optional):
   - Downloads in the last day: 43
   - Downloads in the last week: 181
   - Downloads in the last month: 903

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

I use `!(client.min.js|vendor)` rather than `client*.(css|js|map)` as auto-update rule `npmFileMap/files` under `npmFileMap/basePath:src` to prevent `client.min.js` under `src` from contradicting with the one under `dist` directory and including the files under `src/vendor`.

### CODE REVIEW

**Comment by extend1994** [comment]
@cdnjs/intern3 please help me review this pull request, thank you.

File: ajax/libs/ClientJS/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "description": "Device information and digital fingerprinting written in pure JavaScript.",
+  "keywords": [
+    "browser",
+    "fingerprint",
+    "client",
+    "info",
+    "pure",
+    "javascript"
+  ],
+  "author": "Jack Spirou <jack@spirou.io> (http://twitter.com/jack_spirou)",
```
Comment (sufuf3) [inline ajax/libs/ClientJS/package.json]: use https

**Comment by extend1994** [comment]
@PeterDaveHello please help me review this pull request, thank you.

**Comment by extend1994** [comment]
@PeterDaveHello I updated the rule, please review it, thank you.

**Pull request merged**